chore(dcgw and portal): Add misc FAQ (#5347)
Kong/developer.konghq.com: app/_includes/faqs/dev-portal-graphql.md

{% if include.section == "question" %}

> Can I add GraphQL endpoints to an API or API package, or set GraphQL-specific rate limits, in {{site.dev_portal}}?

Reviewer: ```suggestion
Does {{site.dev_portal}} support adding GraphQL endpoints to an API or API package, or setting GraphQL-specific rate limits?
```
having "Dev Portal" at the end sounds awkward to me (unless "in Dev Portal" is only supposed to apply to the second part, in which case there's just an extra comma.)
